Gordon360/Services/ProfileService.cs

```diff
@@ -23,6 +23,18 @@ public ProfileService(IUnitOfWork unitOfWork)
             _unitOfWork = unitOfWork;
             _accountService = new AccountService(_unitOfWork);
         }
+        /// <summary>
+        /// NOTE: Also a helper method, hence why it returns a StudentProfile model
+        /// rather than a StudentProfileViewModel
+        /// </summary>
+        /// <param name="id">username</param>
```

Comment:
What does it mean that the id is the username? Can you explain a little more?

Reply:
Should be "id", not username. Good Catch!

Reply:
However, I am not using this anymore... I should get rid of this from ProfileService.cs.